services/thermal_mgr/thermal_mgr.c

```diff
@@ -42,18 +42,54 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 }
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
-  /* Send an event to the thermal manager queue */
+  if (thermalMgrQueueHandle == NULL)
+    return ERR_CODE_INVALID_QUEUE_MSG;
```

Comment:
ERR_CODE_INVALID_STATE would be more appropriate here since this isn't really related to the actual queue msg passed in